Api/Config/ConfigInterface.php

```diff
@@ -55,4 +55,46 @@ public function getQueueConsumerInstance($name);
      * @return mixed
      */
     public function getQueueMessageSchema($name);
+
+    /**
+     * Return polling interval in ms
+     *
+     * @return int
+     */
+    public function getQueuePollInterval($name);
+
+    /**
+     * Return maximum number of messages to process
+     *
+     * @return int
+     */
+    public function getQueueLimit($name);
+
+    /**
+     * Return requeue messages on failure
+     *
+     * @return bool
+     */
+    public function getQueueRequeue($name);
```

Comment:
Your code sniffer wants me to rename this is `isQueueRequeue` and same for `isQueueRunOnce`. Please let me know whether you wish for me to make this change.